server/services/prisonServiceHelpers.ts

```diff
@@ -0,0 +1,165 @@
+import { CourtEvent, TransferEvent } from '../@types/prisonApiImport/types'
```

Comment:
No unit tests for this file

Reply:
Yeah this is stuff ive added yesterday after the initial PR for SAA-79. Will add tests

Reply:
Yeah coverage at 61% off the back of the route handler tests. Def needs some tests but didnt want to go overboard as pr Tims alpha steer. Will add some tests

Reply:
Added another test here - but needs some more - will improve in next subtask PR